docs/installation.rst

```diff
@@ -52,6 +52,24 @@ package.
         .. code-block:: console
 
             (.venv) C:\Users\username> python -m pip install -U dffml
+            
+For creating plugin packages you can install git via winget for Windows and Homebrew for MacOS
+
+.. tabs::
+
+    .. group-tab:: MacOS
+
+        .. code-block:: console
+            :test:
```

Comment:
```suggestion
```

Reply:
We don't want to test this quite yet since we don't have any documentation testing systems for macos running (see testing.yml)